DashAI/front/db.json

```
@@ -0,0 +1,191 @@
+{
+  "api": [
+    {
+      "id": 1
+    }
+  ],
+  "dataset": [
+    {
+      "id": 1,
+      "name": "Dataset A",
+      "task_name": "Task A",
+      "created": "2023-04-11T14:30:00",
+      "last_modified": "2023-04-11T14:30:00",
+      "file_path": "/path/to/dataset_A.csv"
+    },
+    {
+      "id": 2,
+      "name": "Dataset B",
+      "task_name": "Task B",
+      "created": "2023-04-11T14:31:00",
+      "last_modified": "2023-04-11T14:31:00",
+      "file_path": "/path/to/dataset_B.csv"
+    }
+  ],
+  "experiment": [
+    {
+      "id": 1,
+      "dataset_id": 1,
+      "task_name": "Task A",
+      "step": "MODEL_CONFIGURATION",
+      "created": "2023-04-11T14:32:00",
+      "last_modified": "2023-04-11T14:32:00"
+    },
+    {
+      "id": 2,
+      "dataset_id": 1,
+      "task_name": "Task A",
+      "step": "EXECUTION",
+      "created": "2023-04-11T14:33:00",
+      "last_modified": "2023-04-11T14:33:00"
+    },
+    {
+      "id": 3,
+      "dataset_id": 2,
+      "task_name": "Task B",
+      "step": "COMPLETED",
+      "created": "2023-04-11T14:34:00",
+      "last_modified": "2023-04-11T14:34:00"
+    }
+  ],
```

```suggestion
  "experiment": [
    {
      "id": 1,
      "name": "Experiment A",
      "dataset_id": 1,
      "task_name": "Task A",
      "step": "MODEL_CONFIGURATION",
      "created": "2023-04-11T14:32:00",
      "last_modified": "2023-04-11T14:32:00"
    },
    {
      "id": 2,
      "name": "Experiment B",
      "dataset_id": 1,
      "task_name": "Task A",
      "step": "EXECUTION",
      "created": "2023-04-11T14:33:00",
      "last_modified": "2023-04-11T14:33:00"
    },
    {
      "id": 3,
      "name": "Experiment C",
      "dataset_id": 2,
      "task_name": "Task B",
      "step": "COMPLETED",
      "created": "2023-04-11T14:34:00",
      "last_modified": "2023-04-11T14:34:00"
    }
  ],
```
I think the names for each experiment are missing.